File: Python/2441.py
n = int(input())
for i in range(n, 0, -1):
    for _ in range(n-i):
        print(' ', end='')
    print('*'*i)


# 인터넷 답안
n=int(input())
for i in range(1,n+1):
    print(" "*(i-1) + "*"*(n-i+1))

File: Archive/My-Baekjoon-online-judge/백준에서 가장 많이 출린 문제 TOP 100/2441.c
#include <stdio.h>
int main() {
	int input;
	scanf("%d", &input);

	int i, j;
	for (i = 0; i < input; i++) {
		for (j = 0; j < i; j++) {
			printf(" ");
		}
		for (j = input; j > i; j--) {
			printf("*");
		}
		printf("\n");
	}
	return 0;
}

/*

*****
 ****
  ***
   **
	*

*/